src/components/manifold-plan-table/manifold-plan-table.scss

```diff
@@ -0,0 +1,510 @@
+/* IBM Plex Mono */
+@use "node_modules/@manifoldco/mercury";
+@import url('https://fonts.googleapis.com/css2?family=IBM+Plex+Mono&display=swap');
+
+// internal component vars
+
+$button-color: mercury.$color-blue;
+$button-text-color: mercury.$color-white;
+$empty-color: mercury.$color-gray;
+$check-color: mercury.$color-blue;
+
+$table-block-spacing: 12px;
+$table-box-shadow: 0 2px 8px rgba(mercury.$color-black 0.0699445);
+$table-border-radius: 6px;
+$table-solid-border: 1px solid mercury.$color-grayLight;
+$table-dashed-border: 1px dashed mercury.$color-grayLighter;
+$table-metered-border: 1px solid #dedede;
+
+$default-color: #dedede;
+$selected-color: mercury.$color-blue;
+$default-chevron-color: #585c5f;
+$transition: 150ms linear;
+
+.mp {
+  @include mercury.Manifold__Typography__Body;
+
+  box-shadow: $table-box-shadow;
+  color: mercury.$color-grayDarker;
+  display: grid;
+  grid-auto-flow: column;
+  grid-template: repeat(var(--table-rows), min-content) / repeat(
+      var(--table-columns),
+      minmax(220px, 250px)
+    );
+  overflow-x: auto;
+
+  & [pattern='[0-9]*'] {
+    font-family: mercury.$typography-monoBodyFontFamily;
+  }
+}
+
+.mp--button {
+  @include mercury.Manifold__Typography__Body;
```

Comment:
Shouldn't this be including the button styles?

Reply:
Working now! 🎉